runner/entrypoint.sh

```diff
@@ -91,6 +91,10 @@ if [ "${RUNNER_FEATURE_FLAG_EPHEMERAL:-}" == "true" -a "${RUNNER_EPHEMERAL}" !=
   config_args+=(--ephemeral)
   echo "Passing --ephemeral to config.sh to enable the ephemeral runner."
 fi
+if [ "${RUNNER_DISABLE_UPDATE:-}" == "true" ]; then
```

Comment:
How is this value passed to the entrypoint.sh? Is that something that should be configured in the runner spec?

In this case, a small bit of documentation would be helpful I guess

Reply:
I guess all the entrypoint environments can be added here? https://github.com/actions-runner-controller/actions-runner-controller#runner-entrypoint-features

Reply:
> Is that something that should be configured in the runner spec?

No, it's excessively complicated to set it in the CRD, we also already configure various runner software flags via environment variables so that is the convention.